nbdev/export2html.py

```diff
@@ -313,6 +313,17 @@ def remove_empty(cells):
 (.*?)$     # Any pattern (shortest possible) then end of line
 """, re.MULTILINE | re.VERBOSE)
 
+_re_mdlinks = re.compile(r"\[(.+)]\((.+)\)", re.MULTILINE)
+
+# Cell
+def _md2html_links(s):
+    'Converts markdown links to html links'
+    matches = _re_mdlinks.findall(s)
+    for name,ref in matches:
```

Comment:
I think this might be faster and cleaner to implement this as a replace? I haven't tested, so it might need to be adjusted but I think just returning
```
_re_mdlinks.sub(r'<a href='\1'>\0</a>, s)
```
should work.

Reply:
Brilliant!!